code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -22,6 +22,10 @@ LINEN BINS
 	dog_fashion = /datum/dog_fashion/head/ghost
 	var/list/dream_messages = list("white")
 
+/obj/item/bedsheet/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/bed_tuckable, 0, 0, 0)
```

Comment:
tucking bedsheets into beds?

Reply:
of course, why not

Reply:
bedsheets go into bed, you don't tuck them under the bedsheets

Reply:
the bed tuckable components checks for a bed, not another bedsheet